modules/ROOT/pages/os-extensions.adoc

```diff
@@ -38,18 +39,27 @@ systemd:
         [Service]
         Type=oneshot
         RemainAfterExit=yes
-        ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive nano
+        # `--allow-inactive` ensure that rpm-ostree does not return an error if
+        # the package is already installed. This is useful if the package is
+        # added to the root image in a future Fedora CoreOS release as it will
+        # prevent the service from failing.
+        ExecStart=/usr/bin/rpm-ostree install --apply-live --allow-inactive vim
         ExecStart=/bin/touch /var/lib/%N.stamp
 
         [Install]
         WantedBy=multi-user.target
 storage:
   files:
-    # use nano as default editor
-    - path: /etc/profile.d/nano.sh
+    # Remove the existing default editor configuration
+    - path: /etc/profile.d/
+      overwrite: true
+      contents:
+        inline: ""
+    # Set vim as default editor
+    - path: /etc/profile.d/vim-default-editor.sh
```

Comment:
WDYT about making it `/etc/profile.d/z-default-editor.sh` so it would work even if we were intalling a different editor.